Cargo.toml

```diff
@@ -15,6 +15,7 @@ rust-version = "1.59.0"
 aes = { version = "0.8.2", optional = true }
 byteorder = "1.4.3"
 bzip2 = { version = "0.4.3", optional = true }
+clap = { version = "4.4.11", features = ["derive"] }
```

Comment:
You should add this as a [`dev-dependency`](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies) instead (as you're only using `clap` for an example).

Reply:
Thank you for pointing this out, I moved `clap` to `dev-dependencies`